python/lsst/pipe/tasks/assembleCoadd.py

```diff
@@ -1446,14 +1446,13 @@ class CompareWarpAssembleCoaddConfig(AssembleCoaddConfig):
         target=SourceDetectionTask,
         doc="Detect outlier sources on difference between each psfMatched warp and static sky model"
     )
-    temporalThreshold = pexConfig.Field(
-        doc="Unitless fraction of number of epochs to classify as an artifact/outlier source versus"
-            " a source that is intrinsically variable or difficult to subtract cleanly. "
-            "If outlier region in warp-diff Chi-image is mostly (defined by spatialThreshold) "
-            "an outlier in less than temporalThreshold * number of epochs, then mask. "
-            "Otherwise, do not mask.",
-        dtype=float,
-        default=0.075
+    maxNumEpochs = pexConfig.Field(
```

Comment:
I would have thought that a fraction would be so much more generally useful than a hard number. I'm worried that a hard number isn't going to scale as you go from n=4 to n=400 inputs.

Reply:
Just made a new ticket for this: DM-12692. I had talked briefly with @RobertLuptonTheGood about it. You're right maxNumEpochs won't scale, but neither will a fraction of the maximum number of epochs.  I've seen some steep Nepoch gradients in a patch.  Its also depends on how many repeat pointings line up the artifacts.

Reply:
Yeah, it probably needs some combination of a number and a fraction, maybe `max(maxNumEpochs, fracMaxEpochs*num)`?